### PR TITLE
Fix tests for categories

### PR DESCRIPTION
**Pull request by Kjaroszew**
I want to merge this change because adding fix for categories

**PR intended to be tested with API branch:** 

### Pull Request Checklist

1. [ ] This code contains UI changes
2. [ ] All visible strings are translated with proper context including data-formatting
3. [ ] Attributes `[data-test-id]` are added for new elements
4. [ ] Changes are mentioned in the changelog
5. [ ] The changes are tested in different browsers and in light/dark mode

### Test environment config

API_URI=https://master.staging.saleor.cloud/graphql/

### CODE REVIEW

File: cypress.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
   "projectId": "51ef7c",
   "chromeWebSecurity": false,
   "videoUploadOnPasses": false,
-  "defaultCommandTimeout": 15000,
+  "defaultCommandimeout": 15000,
```
Comment (eCzerniak) [inline cypress.json:6]: should not remain as it was?
```suggestion
  "defaultCommandTimeout": 15000,
```

File: cypress/integration/catalog/categories.js [hunk 2]
```diff
@@ -1,6 +1,8 @@
 /// <reference types="cypress"/>
 /// <reference types="../../support"/>
 
+import "cypress-wait-until";
```
Comment (eCzerniak) [inline cypress/integration/catalog/categories.js:4]: When I removed it - tests are working but when I leave this import then I cannot execute tests 
```suggestion

```

File: cypress/integration/catalog/categories.js [hunk 3]
```diff
@@ -185,28 +196,40 @@ filterTests({ definedTags: ["all"] }, () => {
...
+            .searchInTable(startsWith)
+            .get(categoryRow(firstCategory.id))
+            .find(BUTTON_SELECTORS.checkbox)
+            .click()
+            .get(categoryRow(secondCategory.id))
+            .find(BUTTON_SELECTORS.checkbox)
+            .click()
+            .get(BUTTON_SELECTORS.deleteIcon)
+            .click()
+            .addAliasToGraphRequest("CategoryBulkDelete")
```
Comment (eCzerniak) [inline cypress/integration/catalog/categories.js:226]: This test is failing - it is too fast ;)
image

File: cypress/support/api/requests/Checkout.js [hunk 4]
```diff
@@ -21,6 +21,7 @@ export function createCheckout({
     billingAddress,
     "billingAddress"
   );
+  cy.log("w funkcji tworzenia");
```
Comment (eCzerniak) [inline cypress/support/api/requests/Checkout.js:24]: Remove it.

File: cypress/support/api/requests/Checkout.js [hunk 5]
```diff
@@ -33,7 +34,7 @@ export function createCheckout({
   );
 
   const emailLine = getValueWithDefault(email, `email: "${email}"`);
-
+  cy.log("przed mutacją");
```
Comment (eCzerniak) [inline cypress/support/api/requests/Checkout.js:37]: Remove it.

File: cypress/support/api/requests/Checkout.js [hunk 6]
```diff
@@ -72,6 +73,8 @@ export function createCheckout({
       }
     }
   }`;
+
+  cy.log(mutation);
```
Comment (eCzerniak) [inline cypress/support/api/requests/Checkout.js:77]: Remove it.

File: package.json [hunk 7]
```diff
@@ -73,6 +73,7 @@
     "react-router-dom": "^5.0.1",
     "react-sortable-hoc": "^1.10.1",
     "react-sortable-tree": "^2.6.2",
+    "saleor-cli": "^0.10.0",
```
Comment (eCzerniak) [inline package.json:76]: Should we change this? Just asking

**Review by eCzerniak — CHANGES_REQUESTED** [review]

File: cypress.json (hunk 1 above)
Comment (Kjaroszew) [reply cypress.json:6]: yes it should

File: cypress/integration/catalog/categories.js (hunk 2 above)
Comment (Kjaroszew) [reply cypress/integration/catalog/categories.js:4]: I forgot to remove import after testing if this library will be helpful to solve problem with reload. This is additional library I've installed so failing in your case was expected.

File: cypress/integration/catalog/categories.js (hunk 3 above)
Comment (Kjaroszew) [reply cypress/integration/catalog/categories.js:226]: this issue will be solved under https://app.clickup.com/t/2549495/SALEOR-6560

File: cypress/support/api/requests/Checkout.js (hunk 4 above)
Comment (Kjaroszew) [reply cypress/support/api/requests/Checkout.js:24]: checkout shouldn't be changed under this task, changes removed

File: cypress/support/api/requests/Checkout.js (hunk 5 above)
Comment (Kjaroszew) [reply cypress/support/api/requests/Checkout.js:37]: checkout shouldn't be changed under this task, changes removed

File: cypress/support/api/requests/Checkout.js (hunk 6 above)
Comment (Kjaroszew) [reply cypress/support/api/requests/Checkout.js:77]: checkout shouldn't be changed under this task, changes removed

File: package.json (hunk 7 above)
Comment (Kjaroszew) [reply package.json:76]: not really, i see it was added automatically to my environment because I'm working with saleor-cli tool

**Review by eCzerniak — APPROVED** [review]

File: cypress/integration/checkout/purchaseWithProductTypes.js [hunk 8]
```diff
@@ -99,6 +99,7 @@ filterTests({ definedTags: ["all", "critical", "refactored"] }, () => {
         billingAddress: address,
         auth: "token"
       })
+        .log({ checkout: checkoutResp })
```
Comment (karola312) [inline cypress/integration/checkout/purchaseWithProductTypes.js:102]: Remove cy.log()

**Review by karola312 — CHANGES_REQUESTED** [review]

**Comment by michalina-graczyk** [comment]
What's the status of this PR? Why it is waiting for so long?

File: cypress/integration/catalog/categories.js [hunk 9]
```diff
@@ -147,23 +151,28 @@ filterTests({ definedTags: ["all"] }, () => {
...
-          .get(BUTTON_SELECTORS.deleteButton)
-          .click()
-          .addAliasToGraphRequest("CategoryDelete")
-          .get(BUTTON_SELECTORS.submit)
-          .click()
-          .waitForRequestAndCheckIfNoErrors("@CategoryDelete");
-        getCategory(categoryResp.id).should("be.null");
-      });
+      visitAddPage();
+      createCategory({ name: categoryName, description: categoryName }).then(
```
[the remark this answers is not in the document]
Comment (Kjaroszew) [reply cypress/integration/catalog/categories.js]: Switch was done mych earlier and i was unable to found who was responsible for that. I've restored old creation. Also with this visitAddPage(); is absolute

**Pull request merged**